pages/publishing/amplitude-govern.mdx

```diff
@@ -0,0 +1,88 @@
+---
+layout: 'guide'
+title: 'Amplitude Govern (API deprecated by Amplitude)'
+# abstract: 'Publishing your tracking plan to Amplitude Govern'
```

Comment:
The `#` marks this as commented-out and it will be discarded when parsed. Is that the intention? If that's the case we could just skip the line altogether.

Reply:
that's not the intetion – thank you for spotting!